package.json

```diff
@@ -11,7 +11,10 @@
     },
     "scripts": {
         "pack": "electron-builder --dir",
-        "start": "node scripts/start.js",
+        "start": "cross-env BUILD_TARGET=web node scripts/start.js",
```

Comment:
To support browser workflow, I added the `BUILD_TARGET` flag. Depending on your workflow, you might want to flip this on its head so that `web` is default and `electron-renderer` is the special case. (On Server, I always run with electron.)